### ISSUE TITLE
Unit type doesn't get expanded into a link on docs

### ISSUE THREAD

**Issue by alvaroc1** [unisonweb/unison#2440]
doc:
```
{{
  Same as {List.map} but returns {type ()}. Mostly useful for executing effects.
}}
```

<img width="406" alt="Screen Shot 2021-09-27 at 8 29 41 AM" src="https://user-images.githubusercontent.com/107395/134928817-5334de0e-6804-4826-86ac-2f80afdb80dc.png">

**Comment by pchiusano** [comment]
Hmm, how about `{type Unit}`?

**Comment by alvaroc1** [comment]
yep, `{type Unit}` works. Should `{type ()}` work too? I always use `()` so i didn't think to try `{type Unit}`.

**Comment by pchiusano** [comment]
It should definitely work! So will leave this open.
